pkg/controller/trustedcabundle/controller.go

```diff
@@ -88,5 +88,5 @@ func (r *ReconcileTrustedCABundle) Reconcile(request reconcile.Request) (reconci
 
 // handleConfigMap returns true if meta namespace is "openshift-logging".
 func handleConfigMap(meta metav1.Object) bool {
-	return meta.GetNamespace() == constants.OpenshiftNS && utils.ContainsString(constants.ReconcileForGlobalProxyList, meta.GetName())
+	return utils.ContainsString(constants.ReconcileForGlobalProxyList, meta.GetName())
```

Comment:
No need for a NS check against the cluster request?

Reply:
same as above